.github/workflows/wheels.yml

```diff
@@ -0,0 +1,65 @@
+name: Build
+
+on: [push, pull_request]
```

Comment:
Would it make sens to have this build and push on pypi done only when we create a release on this project in Github?
I think the syntax would be something like:
```
on:
  release:
    types: [published]
```